docs/features/certificate-authentication.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
Please add a link to the root of the docs and align the name of the file with shared-access-key.
Maybe consider moving it to /features/auth/<>

Reply:
Yes, will do.

Reply:
Done, thanks!